runtime/process_solaris.go

```diff
@@ -0,0 +1,27 @@
+package runtime
+
+import (
+	"golang.org/x/sys/unix"
+	"os"
+	"syscall"
+)
+
+func populateProcessStateForEncoding(config *processConfig, uid int, gid int) ProcessState {
+	return ProcessState{}
+}
+
+func getExitPipe(path string) (*os.File, error) {
+	if err := unix.Mkfifo(path, 0755); err != nil && !os.IsExist(err) {
```

Comment:
@mlaventure : using this for linux fails build. is un../runtime/process_unix.go:7:2: cannot find package "golang.org/x/sys/unix" in any of:
	/usr/local/go/src/golang.org/x/sys/unix (from $GOROOT)
	/go/src/github.com/docker/containerd/vendor/src/golang.org/x/sys/unix (from $GOPATH)
	/go/src/golang.org/x/sys/unix
	/go/src/github.com/docker/containerd/vendor/src/golang.org/x/sys/unix

Did I misunderstand your comment?